requirements_test_all.txt

```diff
@@ -1333,6 +1333,9 @@ soco==0.25.2
 # homeassistant.components.solaredge
 solaredge==0.0.2
 
+# homeassistant.components.solax
+solax==0.2.8
```

Comment:
You will need to change this line to use the new solax library version:
`solax==0.2.9`

After that the diff should be identical. I'll close my duplicate PR

Reply:
Done